src/FINDOLOGIC/Export/CSV/CSVExporter.php

```diff
@@ -56,7 +56,7 @@ public function serializeItemsToFile(
         $csvString = $this->serializeItems($items, $start, $count, $total);
         $targetPath = sprintf('%s/findologic.csv', $targetDirectory);
 
-        file_put_contents($targetPath, $csvString);
+        file_put_contents($targetPath, $csvString, FILE_APPEND);
```

Comment:
Open mode `FILE_APPEND` should only be used if `$start > 0`, otherwise we risk appending to an existing file if one is still in place.

This should also be covered in a test.

Reply:
There should also probably be a test case that an existing file is replaced when using `$start === 0`.